src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const basicPrice = 40 * days;
+
+  if (days < 3) {
+    return basicPrice;
+  }
+
+  if (days < 7) {
+    return basicPrice - 20;
+  }
+
+  return basicPrice - 50;
```

Comment:
let's look at these lines
don't you think you could move those values, 20 and 50, to vars?
by doing so, we simplify the way reviewers understand what's written